diff --git a/CMakeLists.txt b/CMakeLists.txt
index 16ac322..9b4fddf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -777,8 +777,8 @@ if(WITH_PYTHON)
   # Do this before main 'platform_*' checks,
   # because UNIX will search for the old Python paths which may not exist.
   # giving errors about missing paths before this case is met.
-  if(DEFINED PYTHON_VERSION AND "${PYTHON_VERSION}" VERSION_LESS "3.7")
-    message(FATAL_ERROR "At least Python 3.7 is required to build")
+  if(DEFINED PYTHON_VERSION AND "${PYTHON_VERSION}" VERSION_LESS "3.6")
+    message(FATAL_ERROR "At least Python 3.6 is required to build")
   endif()
 
   if(NOT EXISTS "${CMAKE_SOURCE_DIR}/release/scripts/addons/modules")
