diff --git a/aten/src/ATen/cwrap_parser.py b/aten/src/ATen/cwrap_parser.py
index f020dd0..32b4957 100644
--- a/aten/src/ATen/cwrap_parser.py
+++ b/aten/src/ATen/cwrap_parser.py
@@ -15,7 +15,7 @@ def parse(filename):
                 in_declaration = True
             elif line == ']]':
                 in_declaration = False
-                declaration = yaml.load('\n'.join(declaration_lines))
+                declaration = yaml.safe_load('\n'.join(declaration_lines))
                 declarations.append(declaration)
             elif in_declaration:
                 declaration_lines.append(line)
